config/lib/worker.js

```diff
@@ -42,7 +42,7 @@ exports.start = function(options, callback) {
 
     // Schedule job(s)
 
-    agenda.every('5 minutes', 'check unread messages');
+    agenda.every('30 seconds', 'check unread messages');
```

Comment:
Oh yea, I think I had this change somewhere as well just for development.

You recon it would be good to set it this low anyway?